src/towncrier/test/test_check.py

```diff
@@ -13,7 +14,11 @@
 
 def create_project(pyproject_path):
     with open(pyproject_path, "w") as f:
-        f.write("[tool.towncrier]\n" 'package = "foo"\n')
+        f.write(dedent("""\
```

Comment:
I think that we should disable the linter warning for this type of indentation... and maybe just use black :)

I feel that dedent here has no technical merit, and is just added to please a tool :)